multiversxetl/jobs/transform_job.py

```diff
@@ -61,6 +67,10 @@ def _do_transform(self, data: Dict[str, Any]) -> Dict[str, Any]:
         data.pop("nft_traitValues", None)
         data.pop("nft_scamInfoType", None)
         data.pop("nft_scamInfoDescription", None)
+        data.pop("nft_scamInfoVersion", None)
+        data.pop("nft_rarity_score", None)
+        data.pop("nft_rarity_rank", None)
+        data.pop("nft_traitTypes", None)
```

Comment:
Or, perhaps, we should simply remove all fields with the prefix `nft_` instead?

Reply:
we added nft as a prefix to know the service that populate the field, this are added by nft service and when is added by api service we have a prefix api_

Reply:
Fixed, made the transformer a bit more generic.